File: debug_redirect.js
const https = require('https');

const url = 'https://www.akafco.com/pc_software/';

console.log(`Checking URL: ${url}`);

https.get(url, (res) => {
  console.log(`Status Code: ${res.statusCode}`);
  console.log('Headers:');
  for (const [key, value] of Object.entries(res.headers)) {
    console.log(`  ${key}: ${value}`);
  }
  
  if (res.statusCode >= 300 && res.statusCode < 400) {
    console.log(`\nRedirect Location: ${res.headers.location}`);
  }
}).on('error', (e) => {
  console.error(`Error: ${e.message}`);
});

File: debug_redirect.py
import requests

url = "https://www.akafco.com/pc_software/"

try:
    print(f"Checking URL: {url}")
    response = requests.get(url, allow_redirects=False)
    
    print(f"Status Code: {response.status_code}")
    print("Headers:")
    for k, v in response.headers.items():
        print(f"  {k}: {v}")
        
    if response.status_code in (301, 302, 307, 308):
        print(f"\nRedirect Location: {response.headers.get('Location')}")
        
except Exception as e:
    print(f"Error: {e}")
